extensions/amp-sidebar/1.0/test-e2e/test-amp-sidebar.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//
+import {Key} from '../../../../build-system/tasks/e2e/functional-test-controller';
+
+describes.endtoend(
+  'amp-sidebar',
+  {
+    version: '1.0',
```

Comment:
Since the test fixture is being shared, make sure the v0.1 test code also specific `version` in the spec

Reply:
Thanks Ryan!